packages/genesys-spark-components/src/components/stable/gux-tooltip/gux-tooltip.tsx

```diff
@@ -117,7 +118,31 @@ export class GuxTooltip {
   }
 
   private updatePosition(): void {
-    const middleware = [offset(16), flip(), shift(), hide()];
+    const overflowDetection = {
+      name: 'overflowDetection',
+      async fn(state) {
+        const overflowData = await detectOverflow(state, {
+          boundary: 'clippingAncestors',
+          elementContext: 'reference'
+        });
+
+        if (state.placement.includes('bottom') && overflowData.bottom > 0) {
+          return {
+            y: state.y - overflowData.bottom
+          };
+        }
+
+        if (state.placement.includes('top') && overflowData.top > 0) {
+          return {
+            y: state.y + overflowData.top
+          };
+        }
+
+        return {};
+      }
+    };
```

Comment:
`gux-copy-to-clipboard` uses `gux-tooltip` and not the new one. Would it be appropriate to update to `gux-tooltip-beta` with this change (if it's required)? 

Also,  I tested and `gux-tooltip-beta` would require updates similar to the ones I've addd in `gux-tooltip` here for this scenario to be supported.